### PR TITLE
Story#682

### PR DESCRIPTION
**Pull request by deepakr123**

### CODE REVIEW

File: module/users/helper.js [hunk 1]
```diff
@@ -163,4 +163,78 @@ module.exports = class UsersHelper {
...
+   * @param {String} pageNo - page number
+   * @param {String} searchText - search text of users
+   * @param {String} status - status of the users
+   * @param {Array} requestedUsers - array of selected user id
+   * @param {String} email - user email address
+   * @param {String} phone - user phone number
+   * @returns {json} Response consists of users of organisation.
+   */
+
+    static search(token, userId, userName, pageSize, pageNo, searchText, status = "", requestedUsers = [],email="",phone="") {
```
Comment (aks30) [inline module/users/helper.js]: @rakeshSgr - Can we just name it searchByUsername here and in API so its clear no other query will come.

Comment (rakeshSgr) [reply module/users/helper.js]: It's generic code, we can search by name, email and  using other fields as well

File: controllers/v1/users.js [hunk 2]
```diff
@@ -391,4 +391,81 @@ module.exports = class Users {
...
+      try {
+
+        let usersList = await usersHelper.search(
+          req.userDetails.userToken,
+          req.userDetails.userId,
+          req.body.userName ? req.body.userName : "",
+          req.pageSize ? req.pageSize : 1, 
+          req.pageNo ? req.pageNo : 1, 
+          req.body.query ? req.body.query : "",
+          req.body.status ? req.body.status : "",
```
Comment (aks30) [inline controllers/v1/users.js]: @rakeshSgr - Remove page size and query etc, not required just for username search this API is

Comment (rakeshSgr) [reply controllers/v1/users.js]: resolved

**Pull request merged**